test/daily-mode.test.ts

```diff
@@ -122,6 +79,204 @@ describe("Daily Mode", () => {
           { speciesId: SpeciesId.TERAPAGOS, variant: expect.anything(), form: 2, shiny: false },
         ]);
       });
+
+      it("should support custom natures", async () => {
+        vi.spyOn(pokerogueApi.daily, "getSeed").mockResolvedValue(
+          '{"starters":[{"speciesId":1,"nature":0},{"speciesId":113,"nature":1},{"speciesId":1024,"nature":2}]}',
+        );
+        await game.dailyMode.startBattle();
+
+        const natures = game.scene.getPlayerParty().map(p => p.getNature());
+        expect(natures, stringifyEnumArray(Nature, natures)).toEqual([Nature.HARDY, Nature.LONELY, Nature.BRAVE]);
+      });
+
+      describe("Moves", () => {
+        it("should support custom moves", async () => {
+          vi.spyOn(pokerogueApi.daily, "getSeed").mockResolvedValue(
+            '{"starters":[{"speciesId":150,"moveset":[1,2,3,4]},{"speciesId":150,"moveset":[332,6]},{"speciesId":150,"moveset":[130,919]}]}',
+          );
+          await game.dailyMode.startBattle();
+
+          const [moves1, moves2, moves3] = game.scene.getPlayerParty().map(p => p.moveset.map(pm => pm.moveId));
+          expect(moves1, stringifyEnumArray(MoveId, moves1)).toEqual([
+            MoveId.POUND,
+            MoveId.KARATE_CHOP,
+            MoveId.DOUBLE_SLAP,
+            MoveId.COMET_PUNCH,
+          ]);
+          expect(moves2, stringifyEnumArray(MoveId, moves2)).toEqual([
+            MoveId.AERIAL_ACE,
+            MoveId.PAY_DAY,
+            expect.anything(), // make sure it doesn't replace normal moveset gen
+            expect.anything(),
```

Comment:
```suggestion
            expect.any(Number), // make sure it doesn't replace normal moveset gen
            expect.any(Number),
```
I think this should work: https://vitest.dev/api/expect.html#expect-any